2-multitenant/modules/env_baseline/certs.tf

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Create App/Certificates
+resource "google_compute_managed_ssl_certificate" "app_ssl_certificates" {
+  for_each = merge([
+    for key, value in var.apps : {
+      for k, v in value.certificates : "${key}-${k}" => v
+    }
+  ]...)
+
+  name    = each.key
+  project = data.google_project.eab_cluster_project.project_id
+
+  managed {
+    domains = each.value
```

Comment:
how does Google managed cert verify the domain ? the DNS like https://cymbal-bank.endpoints.eab-gke-development-11d0.cloud.goog/  is verifiable by Google managed cert. Other DNS setup may require extra verification steps.

Reply:
Yeah, the current `frontend.example.com` value is basically a placeholder to demonstrate the plumbing, but we will eventually need a value which will work in the CI.

"This resource will "return" as soon as the certificate object is created, but post-creation the certificate object will go through a "provisioning" process. The provisioning process can complete only when the domain name for which the certificate is created points to a target pool which, itself, points at the certificate."

Reply:
ok, sgtm